modules/users/client/views/settings/manage-social-accounts.client.view.html

```diff
@@ -8,8 +8,11 @@ <h3 class="col-md-12 text-center" ng-show="vm.hasConnectedAdditionalSocialAccoun
     </div>
     <div ng-repeat="(providerName, providerData) in vm.user.additionalProvidersData" class="social-account-container">
       <img ng-src="/modules/users/client/img/buttons/{{providerName}}.png">
```

Comment:
It would probably make sense to use [one time binding](https://docs.angularjs.org/guide/expression#one-time-binding) here. (`{{::providerName}}`)